src/Orchard.Cms.Web/Modules/Orchard.Commons/Orchard.Commons.csproj

```diff
@@ -1,16 +1,10 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
+  <Import Project="..\..\..\..\build\common.props" />
```

Comment:
@sebastienros  **common.props** contains general information necessary for assembly